packages/core/src/item/item.js

```diff
@@ -6,6 +6,7 @@ import SearchedItem from "./searched_item/searched_item";
 import BasicItem from "./basic_item";
 
 export const ForwardIconRenderer = () => <>➡️</>;
+const defaultSelectedItem = { item: [], leaf: "" };
```

Comment:
Shouldn't the default be undefined? Nothing selected.

Reply:
Even when nothing is selected I check if selectedItem equals to the current item that renders.
I'm using toString to check if the arrays are equal so I need to give a default that is different them undefined